### ISSUE TITLE
Error when changing datatype with bidseditor GUI on a bidsmap produced with the -s flag

### ISSUE THREAD

**Issue by samuelholly** [Donders-Institute/bidscoin#252]
**Describe the bug**
Hello, as I got the same error as this one #236, I tried to use the `-s` option with `bidsmapper`. When I tried to change the datatype in the bidseditor GUI, I got the following error (user name is redacted):
```
Traceback (most recent call last):
  File "C:\Users\XXX\Anaconda3\envs\qsiprep\Lib\site-packages\bidscoin\bidseditor.py", line 1561, in accept_run
    self.output_bidsmap.update(self.source_run.datatype, self.target_run)
  File "C:\Users\XXX\Anaconda3\envs\qsiprep\Lib\site-packages\bidscoin\bids.py", line 1513, in update
    self.insert_run(runitem)
  File "C:\Users\XXX\Anaconda3\envs\qsiprep\Lib\site-packages\bidscoin\bids.py", line 1475, in insert_run
    targetfile = self.store['target']/sourcefile.relative_to(self.store['source'])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\XXX\Anaconda3\envs\qsiprep\Lib\pathlib.py", line 682, in relative_to
    raise ValueError(f"{str(self)!r} is not in the subpath of {str(other)!r}")
ValueError: 'D:\\DBS_bidsfolder\\code\\bidscoin\\provenance\\tmpbfrlkihe\\DBS_DCM_files\\sub-002\\ses-01\\016-t2_space_da-fl_sag_p3_iso_0.9\\sub-002_t2_space_da-fl_sag_p3_iso_0.9_262.dcm' is not in the subpath of 'C:\\Users\\XXX\\AppData\\Local\\Temp'
``` 

**To reproduce**
1. run `bidsmapper` with `-s` flag
2. Go to bidseditor GUI → click on "Edit" a particular row
3. Click on Datatype → select "anat"
4. Click "OK"
5. bidseditor GUI closes and the above error appears in the cmd window

**Expected behavior**
That BIDS editor will change the datatype

**Screenshots**
NA

**Software version**
 - BIDScoin version: 4.4.0
 - OS: Win10 Pro

**Additional context**
Thank you in advance.

**Comment by marcelzwiers** [comment]
I was able to reproduce and fix (d5997bde3d6b4ccf8a360d4863846487318d9795) your error, thanks for reporting! It was caused by bidsmapper accidentally passing the `-s` provenance store option to the bidseditor. A workaround is to close the (automatically opened) bidseditor and open the bidseditor yourself (by running `bidseditor DBS_bidsfolder`).

**Comment by samuelholly** [comment]
Thank you.
